src/Stratis.Bitcoin.Features.RPC/Stratis.Bitcoin.Features.RPC.csproj

```diff
@@ -36,6 +36,7 @@
 
   <ItemGroup>
     <ProjectReference Include="..\NBitcoin\NBitcoin.csproj" />
+    <ProjectReference Include="..\Stratis.Bitcoin.Features.BlockStore\Stratis.Bitcoin.Features.BlockStore.csproj" />
```

Comment:
Instead of doing this, can you move the BlockModel class to Stratis.Bitcoin?
Normally we should have created a BlockRPCController and put the getblock method to the BlockStore feature, but since we're going to move BlockStore to Stratis.Bitcoin, you might as well just move the BlockModel class and get rid of this reference.